fr.handbrake.ghb.Plugin.dvdcss.json

```diff
@@ -0,0 +1,31 @@
+{
+    "app-id": "fr.handbrake.ghb.Plugin.dvdcss",
+    "branch": "1",
+    "runtime": "fr.handbrake.ghb",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//46",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/app/extensions/dvdcss"
+    },
+    "modules": [
```

Comment:
did this get auto sorted ? It should be module name -> buildsystem -> build-commands -> sources